src/bundlers/node/nodeJsBundler.ts

```diff
@@ -377,11 +379,24 @@ export class NodeJsBundler implements BundlerInterface {
         const mode = input.extra.mode;
         const tmpFolder = input.extra.tmpFolder;
         const cwd = input.projectConfiguration.workspace?.backend || process.cwd();
+        const cloudProvider = input.projectConfiguration.cloudProvider;
 
         if (mode === "development" && !tmpFolder) {
             throw new Error("tmpFolder is required in development mode.");
         }
 
+        var handlerGenerator: (className: string) => string;
+        if (cloudProvider === CloudProviderIdentifier.GENEZIO) {
+            handlerGenerator = lambdaHandlerGenerator;
+        } else if (
+            cloudProvider === CloudProviderIdentifier.CAPYBARA ||
+            cloudProvider === CloudProviderIdentifier.CAPYBARA_LINUX
+        ) {
+            handlerGenerator = genezioRuntimeHandlerGenerator;
+        } else {
+            throw new Error(`Can't generate handler for cloud provider ${cloudProvider}.`);
+        }
```

Comment:
I would use a switch here
```suggestion
        switch(cloudProvider) {
            case CloudProviderIdentifier.GENEZIO:
                 handlerGenerator = lambdaHandlerGenerator;
            case CloudProviderIdentifier.CAPYBARA:
            case CloudProviderIdentifier.CAPYBARA_LINUX:
                 handlerGenerator = genezioRuntimeHandlerGenerator;
            default:
                 throw new Error(`Can't generate handler for cloud provider ${cloudProvider}.`);
        }
```